### PR TITLE
[MRG] New vertical: "Steuerberater" and some updates on the other verticals

### PR DESCRIPTION
**Pull request by michelfernandez**
We refresh the verticals hr and legal and then make a copy of it and it will become the new vertical: Steuerberater

### CODE REVIEW

File: site/content/german/tax-consultant.md [hunk 1]
```diff
@@ -125,10 +125,10 @@ Skribble sowie Partner verwalten ihre Daten gemäss dem Bundesgesetz über den D
 
 {{< content text=center top=24 width=narrow >}}
 {{% markdown %}}
-## Mit Skribble maximieren Sie die Beweiskraft Ihrer Signatur
-Das Gesetz unterscheidet drei Signaturstandards. Sie variieren
+## Alle E-Signatur-Standards aus einer Hand
```
Comment (Kirastina) [inline site/content/german/tax-consultant.md:128]:
```suggestion
## Mit Skribble erhalten Sie alle E-Signatur-Standards aus einer Hand
```

File: site/content/german/tax-consultant.md [hunk 2]
```diff
@@ -125,10 +125,10 @@ Skribble sowie Partner verwalten ihre Daten gemäss dem Bundesgesetz über den D
...
+Das Gesetz unterscheidet drei E-Signatur-Signaturstandards. Sie variieren
 punkto Einsatzgebiet und Beweiskraft. Bei Skribbble erhalten
```
Comment (Kirastina) [inline site/content/german/tax-consultant.md:130]:
```suggestion
punkto Einsatzgebiet und Beweiskraft. 
```

File: site/content/german/tax-consultant.md [hunk 3]
```diff
@@ -125,10 +125,10 @@ Skribble sowie Partner verwalten ihre Daten gemäss dem Bundesgesetz über den D
 
 {{< content text=center top=24 width=narrow >}}
 {{% markdown %}}
-## Mit Skribble maximieren Sie die Beweiskraft Ihrer Signatur
-Das Gesetz unterscheidet drei Signaturstandards. Sie variieren
+## Alle E-Signatur-Standards aus einer Hand
+Das Gesetz unterscheidet drei E-Signatur-Signaturstandards. Sie variieren
 punkto Einsatzgebiet und Beweiskraft. Bei Skribbble erhalten 
-Sie alle Standards aus einer Hand.
+Sie alle.
```
Comment (Kirastina) [inline site/content/german/tax-consultant.md:131]:
```suggestion

```

**Review by Kirastina — COMMENTED** [review]
Duplizierung vom Titel entfernt.

**Comment by TillLord** [comment]
Sorry, I clicked on the review button 🙈 .
I checked the meta title and the meta description and some of the alt texts. It looks very good! Thanks a lot for already implementing the seo parts :)

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

**Tested: OK**

I tested this on every browser (Chrome/FF/Chromium) and on mobile (Native/Chrome) with every language (DE/EN/FR) and it looks good. I found some stuff that needs to be improved afterwards together with something in PR 463.

**Pull request merged**